lib/errorHandlers/suppressError.ts

```diff
@@ -78,10 +99,12 @@ function shouldSuppressError(err, context = {}) {
       subCategory: PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_NOT_SPECIFIED,
     })
   ) {
-    createPlatformVersionError(
-      err.data,
-      PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_NOT_SPECIFIED
-    );
+    if (typeof err === 'object' && err !== null && 'data' in err) {
```

Comment:
Once you update to use import instead of require, you should be able to get rid of these checks